src/components/EReceiptThumbnail.tsx

```diff
@@ -37,30 +35,42 @@ const backgroundImages = {
     [CONST.ERECEIPT_COLORS.PINK]: eReceiptBGs.EReceiptBG_Pink,
 };
 
-function EReceiptThumbnail({transaction}) {
+function EReceiptThumbnail({transaction}: EReceiptThumbnailProps) {
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
-    // Get receipt colorway, or default to Yellow.
-    const {backgroundColor: primaryColor, color: secondaryColor} = StyleUtils.getEReceiptColorStyles(StyleUtils.getEReceiptColorCode(transaction));
 
     const [containerWidth, setContainerWidth] = useState(0);
     const [containerHeight, setContainerHeight] = useState(0);
 
-    const onContainerLayout = (event) => {
+    const backgroundImage = useMemo(() => {
+        if (!transaction) {
+            return;
+        }
+
+        return backgroundImages[StyleUtils.getEReceiptColorCode(transaction)];
+    }, [StyleUtils, transaction]);
```

Comment:
You can turn it back as it was before
```suggestion
const backgroundImage = useMemo(() => backgroundImages[StyleUtils.getEReceiptColorCode(transaction)], [StyleUtils, transaction]);
```

Reply:
Right, updated